en/integrations/stream-connectors/sc-pagerduty.md

```diff
@@ -0,0 +1,158 @@
+---
+id: pagerduty
+title: PagerDuty
+---
+
+## PagerDuty + Centreon Integration Benefits
+
+* Notify on-call system or application administrators when an alert is detected by Centreon.
+* Incidents will automatically resolve in PagerDuty when Centreon detects that the check point is back to normal.
+* Create high and low urgency incidents based on the state of the alert.
+* Send metrics when available to give more insight about the alert.
+
+## How it Works
+
+* Every time a service or a host's state is checked, the event passes through Centreon Broker, which loads the Stream Connector to send state changes.
+* State changes can occur in case of an anomaly detection or metrics falling out of range.
+* Once the check point is back to normal, a resolve event will be sent to the PagerDuty service to resolve the alert.
+
+## Requirements
+
+* PagerDuty integrations require an **Admin base role** for account authorization. If you do not have this role, please reach out to an Admin or Account Owner within your organization to configure the integration.
+* It is also necessary to use a Centreon account with either **admin privileges** or **Export configuration** and **Broker configuration** menu access in the WUI, as well as a **`root` access in command-line interface**.
+
+## Support
+
+If you need help with this integration, depending on how you are using Centreon, you can:
+
+* **Commercial Edition customers**: please contact the [Centreon Support team](mailto:support@centreon.com).
+* **Open Source Edition users** or **Centreon IT-100 users** (free versions): please reach our [Community Slack](https://centreon.github.io) where our users and staff will try to help you.
+
+## Integration Walkthrough
+
+### In PagerDuty
+
+There are two ways to integrate with PagerDuty: via Global Event Routing or on a PagerDuty Service.
+
+If you are adding this integration to an existing PagerDuty service, please skip to the Integrating with a PagerDuty Service section of this guide.
+
+#### Integrating With Global Event Routing
+
+Integrating with Global Event Routing enables you to route events to specific services based on the payload of the event from your tool. If you would like to learn more, please visit [our article on Global Event Routing](https://community.pagerduty.com/forum/t/service-configuration-guide/1660).
```

Comment:
Our article ?
This is pagerduty article right ? not our ?

Reply:
Yes, it was written as this in the template but it's theirs